guide/savings-wallet/miniscript-wallet-simple.md

```diff
@@ -0,0 +1,385 @@
+
+It also suggests that the recovery path will be unlocked after 6 months, if the funds have not been spent. However, users can always choose a different timeframe that better suits their situation.
+
+In addition, our application also provides users with the option to set reminders at different times before the recovery path unlocks.
+
+{% include image-gallery.html pages = page.images_creation-recovery-path %}
+
+Note that the scope of this feature is quite narrow because it is aimed at an audience with basic knowledge of bitcoin. You might want to offer more customization options, based on your own target audience.
+
+#### Save for later
+
+As you can see, creating a multi-key wallet involves a lot of steps in the wallet application as well as on the corresponding signing devices. This can take quite a while, depending on the chosen key scheme. It also requires that users have all extended public keys ready and available beforehand.  
+
+It is likely that not all users will be able or willing to go through the entire process in one setting. This is why our application offers a feature called "Save for later". It allows users to pause the wallet creation process at any time and pick it up later. The application saves the current state of the wallet locally on the device, until it is actually created on-chain.
+
+{% include image-gallery.html pages = page.images_save-for-later %}
+
+#### Wallet backup 
+
+Enabling the recovery path is the final step before the wallet is created on-chain. Once it has been finalized and can be used, our application prompts the user to download the wallet backup file. This is important because, for multi-key wallets, it is not enough to back up the private key to recover the wallet. 
+
+The backup file contains all the information necessary to recover the wallet, except the private keys. This means that it is ok to store the file digitally in a password manager, for example. However, since it contains all extended public keys, anybody who finds this file can potentially monitor all transactions in real time.
+
+Because users might not want to back up the wallet right away, users will also be able to access the backup file and other important information on the wallet settings page at any time. 
+
+{% include image-gallery.html pages = page.images_backup %}
+
+### Co-signer wallet import
+
+There is one final step before Alice and Bob can start using their new savings wallet: Alice nees to import the newly created wallet to the wallet application on her phone. To make this experience easy and seamless, the application allows Bob to share the wallet configuration with Alice in two main ways:
+
+1. By displaying a QR code that Alice scans with her app.  
+2. By sharing the wallet backup file.
+
+{% include image-gallery.html pages = page.images_cosigner-onboarding %}
```

Comment:
> Still don't think you need both images 2 and 3, as they have identical options.

Removed it.

> In the fourth image, maybe put the cancel button in the top-right corner so it isn't accidentally pressed?

I wouldn't have worried about that, since I would use confirmation modals for destructive actions. But I like the idea of having just one immediately visible option 👍

> if you have the bitcoin unit after the number, the Euro symbol should also be after the number.

A keen eye you have 👍. Done as well.